### PR TITLE
feature/nwps_structure

### PR DESCRIPTION
**Pull request by SamiraArdani-NOAA**
<b>Note to developers: You must use this PR template!</b>

## Description of Changes

> Please include a summary of the changes and the related GitHub issue(s). Please also include relevant motivation and context.

This PR initializes the EVS-NWPS, one of the components of EVS that is planned to be added into EVS v2.0. In its final form, EVS-NWPS computes the statistical metrics for verification of nearshore waves and generates the real-time verification graphics. Please note that in this PR, only the "prep" and "stats" step are tested. This PR initializes the EVS-NWPS using forecast files for one WFO (hgx) in southern region. Other WFOs will be added in the future PRs.     

## Developer Questions and Checklist
* Is this a high priority PR? If so, why and is there a date it needs to be merged by?

   No

* Do you have any planned upcoming annual leave/PTO?

   No.

* Are there any changes needed for when the jobs are supposed to run?
  
  No.

- [x] The code changes follow [NCO's EE2 Standards](https://www.nco.ncep.noaa.gov/idsb/implementation_standards/ImplementationStandards.v11.0.0.pdf).
- [x] Developer's name is removed throughout the code and have used `${USER}` where necessary throughout the code.
- [x] References the feature branch for `HOMEevs` are removed from the code.
- [x] J-Job environment variables, COMIN and COMOUT directories, and output follow what has been [defined](https://docs.google.com/document/d/1JWg_4q80aYmmAoD21GFjp9R9y5-3w7WGM3-0HJk0Pjs/edit#heading=h.7ysbr191vzu4) for EVS.
- [x] Jobs over 15 minutes in runtime have restart capability.
- [x] If applicable, changes in the `dev/drivers/scripts` or `dev/modulefiles` have been made in the corresponding `ecf/scripts` and `ecf/defs/evs-nco.def`? 
- [x] Jobs contain the appropriate file checking and don't run METplus for any missing data.
- [x] Code is using METplus wrappers structure and not calling MET executables directly.
- [x] Log is free of any ERRORs or WARNINGs.

## Testing Instructions

> Please include testing instructions for the PR assignee. Include all relevant input datasets needed to run the tests.

1- To test this PR, you need to run the driver script for "prep" and "stats" steps.
2- Clone my fork in your local and checkout branch feature/nwps_structure.
3- ln -s fix directory.
4- In COMIN, point to ${USER}.
5- Run the driver scripts for prep step located at: $EVS/dev/drivers/scripts/prep/nwps/jevs_nwps_wave_grid2obs_prep.sh
6- Run the driver script for stats step located at: $EVS/dev/drivers/scripts/stats/nwps/jevs_nwps_wave_grid2obs_stats.sh

### CODE REVIEW

File: dev/drivers/scripts/plots/glwu/jevs_glwu_wave_grid2obs_plots.sh [hunk 1]
```diff
@@ -61,7 +69,11 @@ export SITE=$(cat /etc/cluster_name)
...
 # CALL executable job script here
 ############################################################
+<<<<<<< HEAD
+${HOMEevs}/jobs/JEVS_GLWU_WAVE_GRID2OBS_PLOTS
+=======
 ${HOMEevs}/jobs/JEVS_GLWU_PLOTS
+>>>>>>> develop
 
 #########################################################################
 # Purpose: This job creates the plots for the NFCENS wave model
```
Comment (PerryShafran-NOAA) [inline dev/drivers/scripts/plots/glwu/jevs_glwu_wave_grid2obs_plots.sh:79]: @SamiraArdani-NOAA In this file you have several of the merge tags (the lines with >>>>>>> and <<<<<<<) that still remain in these files.  Please clean up by removing these tags and ensuring that the lines you want to be part of these scripts remain.

File: dev/drivers/scripts/prep/glwu/jevs_glwu_wave_grid2obs_prep.sh [hunk 2]
```diff
@@ -65,10 +74,17 @@ export SITE=$(cat /etc/cluster_name)
...
+#######################################################################
+# Purpose: This does the prep work for the NFCENS wave model
+=======
 $HOMEevs/jobs/JEVS_GLWU_PREP
 
 #######################################################################
 # Purpose: This does the prep work for the GLWU wave model
+>>>>>>> develop
 # Author: Samira Ardani (samira.ardani@NOAA.gov)
 #######################################################################
```
Comment (PerryShafran-NOAA) [inline dev/drivers/scripts/prep/glwu/jevs_glwu_wave_grid2obs_prep.sh:89]: @SamiraArdani-NOAA You have the merge tags in this file as well.  Please clean up these files by removing the tag.  

Also - why is this file even here?  If this is the nwps PR, you shouldn't have any changes to the glwu file.  Please remove this file from the PR.

File: dev/drivers/scripts/stats/glwu/jevs_glwu_wave_grid2obs_stat.sh [hunk 3]
```diff
@@ -0,0 +1,73 @@
+#PBS -N jevs_glwu_wave_grid2obs_stats
+#PBS -j oe
+#PBS -S /bin/bash
```
Comment (PerryShafran-NOAA) [inline dev/drivers/scripts/stats/glwu/jevs_glwu_wave_grid2obs_stat.sh:3]: @SamiraArdani-NOAA Again, please be sure to remove glwu files from this PR.  That should be throughout this PR.

**Comment by PerryShafran-NOAA** [comment]
@SamiraArdani-NOAA You have many GLWU file changes in this PR.  Please remove them.  This PR only relates to NWPS file changes.  

@AndrewBenjamin-NOAA @malloryprow

**Comment by SamiraArdani-NOAA** [comment]
@PerryShafran-NOAA, Right. I noticed that. They were generated when I was trying to merge develop to this branch. I just received the conflict message along with another message that was saying to use "git add ...". So , I added and committed the changes. The conflict was resolved but those GLWU files changes (already merged to develop) were generated. Do you know how to get rid of those?

**Comment by PerryShafran-NOAA** [comment]
@SamiraArdani-NOAA The best way that I know how is to do the following:

1) Close this PR.
2) Start clean with the current version of develop.
3) Copy in only the files that you changed for nwps.  You may need to create a brand new feature branch to do this.
4) Commit these changes.
5) File a new PR.  

This will ensure we make things the cleanest without going through any difficult steps to add/subtract files from this PR and its feature branch.

**Comment by SamiraArdani-NOAA** [comment]
@PerryShafran-NOAA, Ok. I will do the steps, close this and submit a new PR.

**Pull request closed without merging**